src/components/ha-sidebar.ts

```diff
@@ -117,6 +117,17 @@ class HaSidebar extends LitElement {
             </a>
           `
         )}
+        <a href="#external-app-configuration" tabindex="-1">
```

Comment:
No need to use an `a` tag if we're not using it as an `<a>` ?

Reply:
Without it the button doesn’t match the other buttons.

Reply:
Correction, just checked again. It looks fine but doesn't behave the same. Clicking it leaves it highlighted until clicked away.

![image](https://user-images.githubusercontent.com/18516/56476790-24dbef00-6452-11e9-9826-ce12582ddba5.png)